queries/nu/highlights.scm

```diff
@@ -90,6 +90,10 @@ file_path: (val_string) @variable.parameter
 (escaped_interpolated_content) @string
 (expr_interpolated ["(" ")"] @variable.parameter)
 
+(raw_string_begin) @string
```

Comment:
We probably don't need this, `val_string` will get it properly highlighted.

Reply:
True, removed